toolkit/smart-contracts/offchain/src/init_governance/mod.rs

```diff
@@ -19,8 +19,9 @@ mod tests;
 
 pub(crate) mod transaction;
 
+/// Trait for [init_governance], to make it mockable.
 pub trait InitGovernance {
-	/// Initializes goveranance mechanism with the authority being `governance_authority`,
+	/// Initializes governance mechanism with `governance_parameters`,
```

Comment:
Can we say that it will be `SimpleAtLeastN`  (which in turn explains that it is Cardano Native Script with specific parameters)?